cmake/3rdParty.cmake

```diff
@@ -300,6 +301,10 @@ function(ly_add_external_target)
             )
         endif()
 
+        if(ly_add_external_target_SYSTEM)
+            set_target_properties(3rdParty::${NAME_WITH_NAMESPACE} PROPERTIES LY_SYSTEM_LIBRARY TRUE)
+        endif()
```

Comment:
Can you add a `define_property` for LY_SYSTEM_LIBRARY that can be used to document what the property is used for?

It would be useful later on when someone needs to mark a 3rdParty library has a SYSTEM library that doesn't go through the ly_add_external_target function(Perhaps it was using the regular `find_package` mechanism of cmake) and needs to add that target property manually.

Reply:
Done.